catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveAllCommand.java

```diff
@@ -189,7 +189,8 @@ private Object executeRemoveAllFromStore() throws Exception {
             console.flush();
 
             // Break out if there are no more records to delete
-            if (amountDeleted < batchSize || batchSize < 1) {
```

Comment:
I added the `batchsize < 1` back in as it does not break the tests this PR is focusing on. @lessarderic might this be part of a future task with removing the magic numbers? I feel for the time being we'll leave it in and later change if it's deemed unnecessary.